src/HttpServer.php

```diff
@@ -17,20 +17,26 @@ class HttpServer extends EventEmitter
 
 	private $socket;
 	private $callback;
+	private $errorResponse;
 
 	/**
 	 *
 	 * @param ServerInterface $socket - the server runs on this socket (ip address and port)
 	 * @param callable $callback - callback function which returns a RingCentral\Psr7\Response
 	 */
-	public function __construct(ServerInterface $socket, callable $callback)
+	public function __construct(ServerInterface $socket, callable $callback, Response $errorResponse = null)
```

Comment:
Thank you for your comment :grin: I added an example to make the purpose a little clearer. The error response message will be thrown, when an uncaught exception occures in the callback function of the user.

You can still react on responses with any request you want in your own user code in the callback function.

Because of this "feature" the server isn't able to crash so easily by uncaught exceptions.